### ISSUE TITLE
post to getlektor.com

### ISSUE THREAD

**Issue by nixjdm** [JavierLopezMunoz/lektor-google-tag-manager#1]
Hey @JavierLopezMunoz! I'm Lektor's maintainer, and I found your plugin. I want to encourage you to open a pull request to https://github.com/lektor/lektor-website to add it to the list of available plugins. You can read about how to do that here: https://www.getlektor.com/docs/plugins/publishing/

I hope you do!

Thanks!

**Comment by JavierLopezMunoz** [comment]
Hi @nixjdm, sure. I will very happy that the plugging is listed in the official Lektor webpage.

Shall I add it here https://www.getlektor.com/plugins/? In which section shall I add it?

Best

**Comment by nixjdm** [comment]
Yes, it should be a subpage of https://www.getlektor.com/plugins/. getlektor will regularly pull information from pypi, so your plugin's setup.py should be filled out and then a new version published to pypi.org. You can copy a lot from the example in https://www.getlektor.com/docs/plugins/publishing/
